gitlab/resource_gitlab_project_mirror.go

```diff
@@ -0,0 +1,170 @@
+package gitlab
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/xanzy/go-gitlab"
+)
+
+func resourceGitlabProjectMirror() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGitlabMirrorCreate,
+		Read:   resourceGitlabProjectMirrorRead,
+		Update: resourceGitlabMirrorUpdate,
+		Delete: resourceGitlabMirrorDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"project_id": {
```

Comment:
"project" seems to be more common in this provider than "project_id", although I do prefer "project_id" myself. Is that a conscious choice?

Reply:
I copied the schema from gitlab project member, so not conscious. Happy to change